rpc-jobs/RPC-AIO.yml

```diff
@@ -19,12 +19,21 @@
           name: IMAGE
           default: "Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)"
       - string:
+          name: INSTANCE_NAME
+          default: AUTO
+          description: |
+            Set instance name or "AUTO" to have it generated based on job
+            name and build number. Beware the 63 char container name limit
+            as container names includes the host name.
+      - string:
           name: STAGES
-          default: "Allocate Resources, Cleanup"
+          default: "Allocate Resources, Connect Slave, Pause, Cleanup"
```

Comment:
If we pause by default, will that turn into a manual burden of having to input continue for all the builds?